src/handler/BallotHandler.ts

```diff
@@ -30,17 +32,12 @@ export class BallotHandler {
     await ballotService.deleteBallot(ballotID);
   }
 
-  public getBallots = ballotService.getBallots;
-
   public async getRunningBallot() {
     const ballot = await ballotService.getRunningBallot();
-
-    if (!ballot) {
-      throw new NotFound("There is no running ballot");
-    }
+    if (!ballot) throw new NotFound("There is no running ballot");
```

Comment:
You're walking on some thin ice here. I already got flamed twice for omitting the brackets by some contributors :(.
But I also prefer doing it this way.

Reply:
Its our backend now we define the rules. 
(▀̿Ĺ̯▀̿ ̿)

Reply:
I think its fine to omit them :)